utils.go

```diff
@@ -212,6 +212,16 @@ func (h *Headscale) getUsedIPs() ([]netaddr.IP, error) {
 	return ips, nil
 }
 
+func containsString(ss []string, s string) bool {
```

Comment:
Man, I am looking forward to Generics...

Reply:
So am I :smile: